nullaway/src/main/java/com/uber/nullaway/generics/GenericsChecks.java

```diff
@@ -768,6 +884,51 @@ public static Nullness getGenericParameterNullnessAtInvocation(
       MethodInvocationTree tree,
       VisitorState state,
       Config config) {
+    List<? extends Tree> typeArgumentTrees = tree.getTypeArguments();
```

Comment:
Same comment as above regarding only running the new logic when a generic method is being invoked